src/data_inclusion/scripts/tasks/sources/mes_aides.py

```diff
@@ -0,0 +1,119 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytz
+from dateutil.parser import parse as dateutil_parse
+from pyairtable import Table
+
+from data_inclusion.schema import models
+
+MES_AIDES_SOURCE_STR = "mes-aides"
+
+
+def extract_data(api_key: str, base_id: str, table_name: str) -> Path:
+    dt = datetime.now(tz=pytz.UTC).isoformat(timespec="seconds")
+    output_path = Path(f"./garages_solidaires.{dt}.json")
+
+    table = Table(api_key=api_key, base_id=base_id, table_name=table_name)
+    table.all()
+
+    data = [col["fields"] for col in table.all()]
+    pd.DataFrame(data).to_json(output_path, orient="records", force_ascii=False)
+
+    return output_path
+
+
+def transform_data(path: Path) -> Path:
+    output_path = Path(f"./{path.stem}.reshaped.json")
+    input_df = pd.read_json(path, dtype=False).replace(np.nan, None)
+    output_df = transform_dataframe(input_df)
+    output_df.to_json(output_path, orient="records", force_ascii=False)
+    return output_path
+
+
+def transform_dataframe(input_df: pd.DataFrame) -> pd.DataFrame:
+    input_df = pd.json_normalize(input_df.to_dict(orient="records"))
+    input_df = input_df.replace("", None)
+    output_df = pd.DataFrame()
+
+    # id
+    output_df = output_df.assign(id=input_df["ID"])
+
+    # siret
+    # No siret available in data source
+    output_df = output_df.assign(siret=None)
+
+    # rna
+    output_df = output_df.assign(rna=None)
+
+    # nom
+    output_df = output_df.assign(nom=input_df["Nom"])
+
+    # commune
+    output_df = output_df.assign(commune=input_df["Ville Nom"])
+
+    # code_postal
+    output_df = output_df.assign(code_postal=input_df["Code Postal"])
+
+    # code_insee
+    output_df = output_df.assign(code_insee=None)
+
+    # adresse
+    output_df = output_df.assign(adresse=input_df["Adresse"])
+
+    # complement_adresse
+    output_df = output_df.assign(complement_adresse=None)
+
+    # longitude
+    output_df = output_df.assign(longitude=input_df["Ville Longitude"])
+
+    # latitude
+    output_df = output_df.assign(latitude=input_df["Ville Latitude"])
+
+    # typologie
+    output_df = output_df.assign(typologie=models.Typologie.Autre.value)
+
+    # telephone
+    output_df = output_df.assign(telephone=input_df["Téléphone"])
+
+    # courriel
+    output_df = output_df.assign(courriel=input_df["Email"])
+
+    # site_web
+    output_df = output_df.assign(site_web=input_df["Url"])
+
+    # presentation_resume
+    output_df = output_df.assign(presentation_resume=None)
+
+    # presentation_detail
+    output_df = output_df.assign(presentation_detail=None)
+
+    # source
+    output_df = output_df.assign(source=MES_AIDES_SOURCE_STR)
+
+    # date_maj
+    output_df = output_df.assign(
+        date_maj=input_df["Modifié le"].map(lambda s: dateutil_parse(s).isoformat())
+    )
+
+    # structure_parente -> CRITERE A VALIDER
+    output_df = output_df.assign(structure_parente=input_df["Partenaire Nom"])
```

Comment:
```suggestion
    output_df = output_df.assign(structure_parente=None)
```
Ce champs référence le champs `id` d'une autre ligne du jeu de données.
Il y a peut-être des antennes dans les données, mais leur schéma ne permet pas de retrouver ce lien facilement.
On peut laisser à la valeur nulle pour le moment